app/components/QuestionRow/QuestionRow.jsx

```diff
@@ -131,21 +150,32 @@ function QuestionRow(props) {
           {profile.is_admin ? adminPinButton : nonAdminPinIndicator}
           {profile.is_admin && (
 
-            <Styled.DisableControls>
-              <Styled.ButtonTooltipMessage>
-                Click to
-                {' '}
-                {question.is_enabled ? 'disable' : 'enable'}
-                {' '}
-                this
-                question
-              </Styled.ButtonTooltipMessage>
-              <Switch
-            id={`question-${question.question_id}`}
-            checked={question.is_enabled}
-            onChange={handleStatusClick}
-          />
-            </Styled.DisableControls>
+            <>
+              <Styled.DisableControls>
+                <Styled.ButtonTooltipMessage>
+                  Click to
+                  {' '}
+                  {question.is_enabled ? 'disable' : 'enable'}
+                  {' '}
+                  this
+                  question
+                </Styled.ButtonTooltipMessage>
+                <Switch
+                  id={`question-${question.question_id}`}
+                  checked={question.is_enabled}
+                  onChange={handleStatusClick}
+                />
+              </Styled.DisableControls>
+              {(!question.is_public && question.is_enabled) && (
+                <Button
```

Comment:
Maybe we can add a tooltip to the button for clarity?

Reply:
Done ✅